src/rt/GEAR/rt_grackle_utils.h

```diff
@@ -20,7 +20,7 @@
 #define SWIFT_RT_GRACKLE_UTILS_H
 
 /* skip deprecation warnings. I cleaned old API calls. */
-#define OMIT_LEGACY_INTERNAL_GRACKLE_FUNC
```

Comment:
why?

Reply:
Since when enable the subgrid physics, this macro is defined somewhere already. The error suggests I can delete this definition